py-rse/git-cmdline.Rmd

```diff
@@ -20,8 +43,8 @@ Using this syntax,
 the first thing we need to do is configure Git:
 
 ```shell
-$ git config --global user.name "Frances Bilas"
-$ git config --global user.email "frances@eniac.org"
+$ git config --global user.name "Amira Lastname"
+$ git config --global user.email "amira@zipfs.org"
 ```
```

Comment:
Maybe we need to give our fictional characters in Section 1.1 a last name?

Reply:
Yup.  Asking for suggestions in #340